data/remnant/remnant outfits.txt

```diff
@@ -662,6 +662,7 @@ outfit "Salvage Scanner"
 	"outfit scan power" 13
 	"outfit scan efficiency" 12
 	"tactical scan power" 84
+	"maneuver scan power" 100
```

Comment:
and... no, you should do this in another PR, after this one got merged. 

This PR should only include discussions about the feature and not discussion about the implementation.
I know I am annoying about this point, but I will keep reminding people of this.

Reply:
Fine, I'll include it in the Remnant Scan Fix PR.